handlers/register.go

```diff
@@ -0,0 +1,33 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/companieshouse/chs.go/authentication"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/emergency-auth-code-api/config"
+	"github.com/gorilla/mux"
+)
+
+// Contains config and DAO for
+type EmergencyAuthCodeService struct {
+	Config *config.Config
+}
+
```

Comment:
Exported function needs a comment
```suggestion
// Register defines the endpoints for the API
```